kazoo_translator/src/converters/kzt_twiml_util.erl

```
@@ -60,13 +60,16 @@ get_voice(Props) ->
 get_engine(Call, Props) ->
     case props:get_ne_binary_value('engine', Props) of
         'undefined' -> ?DEFAULT_TTS_ENGINE(Call);
-        Engine -> Engine
+        Engine ->
+          case lists:member(Engine, ?DEFAULT_SUPPORTED_PROVIDERS) of
```

No need for using macros. Also adding default is really misleading :) At first read i thought you were checking `Engine` is in the list of default providers.

```suggestion
          case lists:member(Engine, kazoo_tts:supported_providers()) of
```